test/functional/pages_controller_test.rb

```diff
@@ -8,6 +8,5 @@ def setup
   test 'should get how_to' do
     get :show, params: { id: 'how_to' }
     assert_response :success
-    assert_template :how_to
```

Comment:
Really? Is there a replacement? Do you know why they don't support it anymore?

Reply:
I added a commit where I replaced these with assertions that check the response body.